src/sqs/backfill.ts

```diff
@@ -5,8 +5,9 @@ import * as Sentry from "@sentry/node";
 import { AxiosErrorEventDecorator } from "models/axios-error-event-decorator";
 import { SentryScopeProxy } from "models/sentry-scope-proxy";
 import { TaskType } from "~/src/sync/sync.types";
+import { GitHubAppConfigPayload } from "./github-app-config-payload";
 
-export type BackfillMessagePayload = {
+export type BackfillMessagePayload = GitHubAppConfigPayload & {
```

Comment:
This syntax is new to me. Is this adding the attributes below to the type `GitHubAppConfigPayload`?

Reply:
it's the same as `interface BackfillMessagePayload extends GitHubAppConfigPayload { ... }`, but done with the `type` syntax instead of `interface`

Reply:
Closing it, since not using this syntax in the PR now.